invenio_records_rest/views.py

```diff
@@ -69,6 +67,12 @@ def create_blueprint(endpoints):
         for rule in create_url_rules(endpoint, **options):
             blueprint.add_url_rule(**rule)
 
+    # catch record validation errors
+    @blueprint.errorhandler(ValidationError)
+    def validation_error(error):
+        """Catch validation errors."""
+        return "Validation error", 400
```

Comment:
We should be careful with response body. We should probably return body of a  `RESTException`.